_drafts/_heat/Heat_Orchestration_Template_LB.yaml

```diff
@@ -0,0 +1,109 @@
+heat_template_version: 2013-05-23
+description: Simpler server with lb
+parameters:
+  image:
+    type: string
+    description: Image used for servers
+    default: Red Hat Enterprise Linux 6.5 x86_64
```

Comment:
This image requires a license. We should re-vamp this example to use a free use operating system.